common/common.mk

```diff
@@ -1,89 +0,0 @@
-## Using ?= makes these variables overridable
-## Simply define them before including common.mk
-CXX := clang++-10
-STDCXX ?= c++17
-CFLAGS := $(CFLAGS) -DGLSL_VERSION='"330 core"'
-RM := rm -f
-
-## A compilation flag for selecting the Monado integration mode
-#> If ILLIXR_MONADO_VULKAN is set to 'ON', disable OpenGL initializatin
-#> Else ILLIXR_MONADO_VULKAN is empty, enabling the monado_integration compatible compilation
-ifeq ($(ILLIXR_MONADO_VULKAN),ON)
-	MONADO_FLAGS := -DILLIXR_MONADO_MAINLINE
```

Comment:
I wasn't able to find where this is defined in the Cmake files. This needs to be set for both the gl and vk flavors of Monado.

Reply:
This may be an adjustment to the OpenGL rebase, which defines `ILLIXR_MONADO` and `ILLIXR_VULKAN` as separate defines.

Reply:
Once the OpenGL rebase is merged into main its changes will be reflected in this branch.

Reply:
The addition of the -DILLIXR_MONADO_MAINLINE flag is now part of the build system.